pkg/core/resource_graph_test.go

```diff
@@ -96,8 +104,9 @@ func TestResourceGraph_AddDependenciesReflect(t *testing.T) {
 		"value_arr1", "value_arr2",
 		"value_ptr_arr1", "value_ptr_arr2",
 		"value_map1", "value_map2",
-		"value_ptr_map1", "value_ptr_map2",
+		"value_ptr_map1", "value_ptr_map2", "nested",
```

Comment:
nit: these were generally only on the same line if they were related - since this one isn't related to anything it should get its own line